client/packages/invoices/src/OutboundShipment/DetailView/SidePanel/PricingSection.tsx

```diff
@@ -73,7 +77,7 @@ const ServiceCharges = ({ pricing, isDisabled }: PricingGroupProps) => {
         <PanelLabel>{`${t('heading.tax')} ${Formatter.tax(tax)}`}</PanelLabel>
         <PanelField>
           <TaxEdit
-            disabled={!serviceLines?.length || isDisabled}
+            disabled={disableServiceTax || isDisabled}
```

Comment:
Is not `disableServiceTax` would be the same if True but `isDisabled` is False?

Reply:
`isDisabled` is just checking if the invoice data exists... So will always be true if invoice is in DB... The pen will always be disabled if service total is 0, regardless if isDisabled is false since it's checking OR...
Is that what you mean?